### PR TITLE
Dockarized app

### PR DESCRIPTION
**Pull request by tonidezman**
Adds quick start up for people that use docker and docker-compose

### CODE REVIEW

**Comment by brandon-rhodes** [comment]
@tonidezman — Thank you! I'll be busy this weekend but should have a chance to look at this and comment early next week.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM continuumio/miniconda3:4.9.2-alpine
```
Comment (brandon-rhodes) [inline Dockerfile]: Could these docker files go in a different directory from the main project directory? I'd prefer not to have tool-specific files up at the top level. Thanks!

File: README.md [hunk 2]
```diff
@@ -18,16 +18,28 @@ under the MIT license (see the accompanying `LICENSE.txt` file) and I
 have tried to make sure that this repository contains all of the scripts
 needed to download and set up the data set that we used.
 
-## Quick Start
+## Quick Start (🐳 Docker)
+```bash
+# clone and go inside the repo
```
Comment (brandon-rhodes) [inline README.md]: Could you pivot away from these comments, and match the sections above and below this one by using paragraphs and full sentences here in this section?

**Review by brandon-rhodes — COMMENTED** [review]
Okay, I've had time to read over this. Thanks for working to make the tutorial easier for folks to use! I've made a couple of comments.

File: Dockerfile (hunk 1 above)
Comment (tonidezman) [reply Dockerfile]: Hi Brandon. Sure I will move both docker related files to build directory.

Comment (tonidezman) [reply Dockerfile]: @brandon-rhodes  I have a problem. When I add the docker related files the exercise files are not accessible. I have tried:
- changing dockerfile copy path
- changing context in build stage in docker-compose
with no luck.

You said that you like to move the build files to other directories I would like to know how you would fix this?

Comment (tonidezman) [reply Dockerfile]: current copied files

Comment (brandon-rhodes) [reply Dockerfile]: For the XEphem project, I have a directory `Containers` with the various Docker files inside — which lets me maintain several Docker files targeting different container operating systems.

https://github.com/XEphem/XEphem

Could you see whether a similar approach would work here?

File: README.md (hunk 2 above)
Comment (tonidezman) [reply README.md]: Thanks for the link Brandon. It helped me a lot :)

**Pull request closed without merging**